optin/reporting/test/utils/ServerUtils.spec.js

```diff
@@ -0,0 +1,170 @@
+const Utils = require('../../utils/ServerUtils');
+const { expect } = require('chai');
+
+describe('server utils', () => {
+	describe('toRelativeAmount', () => {
+		it('return relative amount given string integer', () => {
+			// Arrange:
+			const amount = '1';
+
+			// Act:
+			const relativeAmount = Utils.toRelativeAmount(amount);
+
+			// Assert:
+			expect(relativeAmount).to.be.equal(0.000001);
+		});
+
+		it('return relative amount given integer', () => {
+			// Arrange:
+			const amount = 1;
+
+			// Act:
+			const relativeAmount = Utils.toRelativeAmount(amount);
+
+			// Assert:
+			expect(relativeAmount).to.be.equal(0.000001);
+		});
+
+		it('return undefined given alphabet string', () => {
+			// Arrange:
+			const amount = 'xym';
+
+			// Act:
+			const relativeAmount = Utils.toRelativeAmount(amount);
+
+			// Assert:
+			expect(relativeAmount).to.be.equal(undefined);
+		});
+	});
+
+	describe('byteToHexString', () => {
+		it('return hex given uint 8 array', () => {
+			// Arrange:
+			const byte = new Uint8Array([
+				104, 4, 35, 192, 130, 28, 131,
+				242, 247, 23, 168, 248, 242, 123,
+				25, 204, 23, 78, 212, 211, 161,
+				220, 59, 4, 48
+			]);
+
+			// Act:
+			const result = Utils.byteToHexString(byte);
+
+			// Assert:
+			expect(result).to.be.equal('680423C0821C83F2F717A8F8F27B19CC174ED4D3A1DC3B0430');
+		});
+
+		it('return hex given string', () => {
+			// Arrange:
+			const byte = '1234';
+
+			// Act:
+			const result = Utils.byteToHexString(byte);
```

Comment:
does `byteToHexString`accept strings instead of byte arr? If so should we state it in the function name?